docs/TODO.md

```diff
@@ -45,6 +45,11 @@ This was previously marked 'done' but is now very much "un-done" since the code
 
 * The [issue693](https://github.com/JoinMarket-Org/joinmarket/issues/693) problem is by far the most important one to spend time on.
 
+Possible future update ("hard fork") for Joinmarket protocol may include:
+* bech32 addresses for joins; possibly mixed address types, needs discussion.
+* change format of data transfer (in particular, avoid double encoding which wastes space)
+* w.r.t above issue693, see fidelity bonds [discussion](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-August/017204.html) ; if implemented this may (will?) require some changes to protocol messages.
```

Comment:
Maybe not so important, but w.r.t. was an abbreviation I needed google for as a non-native English speaker... :)

Also, maybe adding link to [#371](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/371) here would not hurt.

Reply:
Yes, thanks for the feedback, and I'll use that link instead of the mailing list one